### PR TITLE
Use new serializers in NetSuite

### PR DESCRIPTION
**Pull request by ryanwjackson**

### CODE REVIEW

File: lib/ledger_sync/adaptors/netsuite/customer/serializer.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+          attribute :phone,
+                    resource_attribute: :phone_number
+
+          # attribute :firstName,
+          #           resource_attribute: :first_name
+
+          # attribute :lastName,
+          #           resource_attribute: :last_name
+
```
Comment (SeanBolt) [inline lib/ledger_sync/adaptors/netsuite/customer/serializer.rb:27]: Are we leaving these commented out blocks in?

File: lib/ledger_sync/adaptors/netsuite/adaptor.rb [hunk 2]
```diff
@@ -60,6 +60,14 @@ def get(**keywords)
           request(keywords.merge(method: :get))
         end
 
+        def ledger_resource_path(args = {})
+          resource = args.fetch(:resource, nil)
+
+          ret = resource.class.resource_type.to_s # This can be turned into a case statement if we need to override
```
Comment (SeanBolt) [inline lib/ledger_sync/adaptors/netsuite/adaptor.rb:66]: How will we deal with `ledger_resource_type_overrides`?

File: lib/ledger_sync/adaptors/netsuite/customer/serializer.rb (hunk 1 above)
Comment (ryanwjackson) [reply lib/ledger_sync/adaptors/netsuite/customer/serializer.rb:27]: These were in the originals, so I left them.  Not sure why though.  I figured I'd leave them for now until we decide to add/remove.

File: lib/ledger_sync/adaptors/netsuite/adaptor.rb (hunk 2 above)
Comment (ryanwjackson) [reply lib/ledger_sync/adaptors/netsuite/adaptor.rb:66]: For now, we are going to have to just override here as needed with a case. The issue is that the current overrides only considers the class.  But the NS item for example will switch depending on a value of the item.

I'd say lave and then we perhaps combine concepts as we figure out exactly how to handle the more polymorphic relationships.

**Pull request merged**